cpython/Makefile

```diff
@@ -79,11 +88,23 @@ $(ZLIBBUILD)/.patched: $(ZLIBTARBALL)
 	touch $@
 
 
-$(BUILD)/Makefile: $(BUILD)/.patched $(ZLIBBUILD)/.patched
+$(SQLITEBUILD): $(SQLITETARBALL)
```

Comment:
Using a directory as a make target doesn't always work correctly. It's better to point to a file that is created by the sqlite build process to detect when it is done.  Maybe $(SQLITEBUILD)/libsqlite3.la ?

Reply:
Thanks. Resolved in 1947e44.